extensions/2.0/Khronos/KHR_materials_transmission/examples/transmission_example.gltf

```diff
@@ -0,0 +1,1117 @@
+            "mimeType": "image/jpeg"
+        }
+    ],
+    "materials": [
+        {
+            "pbrMetallicRoughness": {
+                "baseColorTexture": {
+                    "index": 0
+                },
+                "metallicFactor": 0.0,
+                "roughnessFactor": 0.4934999942779541
+            },
+            "name": "Cloth Backdrop"
+        },
+        {
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [
+                    0.9559729695320129,
+                    0.17788879573345185,
+                    0.09530720859766007,
+                    1.0
+                ],
+                "metallicFactor": 0.0,
+                "roughnessFactor": 0.0
+            },
+            "alphaMode": "BLEND",
+            "name": "RedTransTexture",
+            "doubleSided": true,
+            "extensions": {
+                "KHR_materials_transmission": {
+                    "ior": 1.7999999523162842,
+                    "transmissionTexture": {
+                        "index": 1
+                    }
+                }
+            }
+        },
+        {
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [
+                    0.8468738198280335,
+                    0.8713662624359131,
+                    0.09305903315544129,
+                    1.0
+                ],
+                "metallicFactor": 0.0,
+                "roughnessFactor": 0.0
+            },
+            "alphaMode": "BLEND",
+            "name": "YellowTrans",
+            "doubleSided": true,
+            "extensions": {
+                "KHR_materials_transmission": {
+                    "ior": 1.7999999523162842,
+                    "transmissionFactor": 1
+                }
+            }
+        },
+        {
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [
+                    0.08650036156177521,
+                    0.46207714080810549,
+                    0.8468738198280335,
+                    1.0
+                ],
+                "baseColorTexture": {
+                    "index": 2,
+                    "extensions": {
+                        "KHR_texture_transform": {
+                            "offset": [
+                                0,
+                                -2.1010000705718996
+                            ],
+                            "rotation": 0,
+                            "scale": [
+                                3.1010000705718996,
+                                3.1010000705718996
+                            ]
+                        }
+                    }
+                },
+                "metallicFactor": 0.0,
+                "roughnessFactor": 0.0
+            },
+            "alphaMode": "BLEND",
+            "name": "BlueTransWithMask",
+            "doubleSided": true,
+            "extensions": {
+                "KHR_materials_transmission": {
+                    "ior": 1.7999999523162842,
```

Comment:
is `ior` part of this?

Reply:
Oops. That's left over from some previous work. No, this extension will use the surface IOR defined by either the core spec or another extension.

Reply:
Thanks for pointing that out. I've removed them.